config/local/catalog-standalone-defs/apiDocs/service1-instance1_org.zowe v1.0.0_default.json

```diff
@@ -0,0 +1,1225 @@
+{
```

Comment:
is the content identical in all 4 docs?

Reply:
Almost, the differences should be used in integration tests. But I'll make them smaller.

Reply:
Is this one pending too?